ynput/OpenPype
# Maya: do not create top level group on reference

## Changelog Description
This PR allows to not wrapping loaded referenced assets in top level group either explicitly for artist or by configuration in Settings.
Artists can control group creation in ReferenceLoader options.
Default no group creation could be set by emptying `Group Name` in `project_settings/maya/load/reference_loader`


## Testing notes:
1. try to uncheck `Group imported assets` in Reference Loader options
![AnyDeskMSI_XPqOsgRKGD](https://github.com/ynput/OpenPype/assets/4457962/370183bc-7ff5-4edf-8c91-4803378500d0)
3. no group should be created
4. remove group name in `project_settings/maya/load/reference_loader` >> no need to go to Options, still no group will be created
![V9nxVMBye8](https://github.com/ynput/OpenPype/assets/4457962/68b37e3e-9e2b-458d-94ee-049454f42cff)

![AnyDeskMSI_mo3PDISCHN](https://github.com/ynput/OpenPype/assets/4457962/5227af70-50ed-45f1-b95f-6095764b0421)
